src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingEnchanter.java

```diff
@@ -33,7 +34,7 @@
 /**
  * The enchanter building.
  */
-public class BuildingEnchanter extends AbstractBuildingWorker
+public class BuildingEnchanter extends AbstractBuildingCrafter
```

Comment:
This building was turned into a crafter. Yet no methods where overriden. Sooo is this a default crafter? I think you would like ant to limit the recipes would you not?

Reply:
@OrionDevelopment afaik that is all data driven now. @Mekle001 this should be fine as is, right?

Reply:
hm good point I'll go ahead and test it

Reply:
was able to teach it sticks xD